.github/workflows/guitest.yml

```diff
@@ -25,10 +25,14 @@ jobs:
       fail-fast: false
       matrix: ${{fromJson(inputs.matrix)}}
 
+    defaults:
+      run:
+        shell: bash
```

Comment:
This is the fix for Windows GUI tests. 
Without it, the wrong env had been used and this led to the execution of the whole test suite.